templates/simple/flake.nix

```diff
@@ -35,7 +34,7 @@
           module = config;
           # You can use `extraSpecialArgs` to pass additional arguments to your module files
           extraSpecialArgs = {
-            inherit self;
+            # inherit (inputs) foo;
```

Comment:
Why leave this as a comment? It's to show how to pass something?

Reply:
Yes, it is just to show that it can be done.